test/rostest_avoidance_run.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
```

Comment:
This is fine for now, but we should try to think of a better way to do this. For example having it locally and running repeatedly shouldn't have to clone and build each time.